include/functions.h

```diff
@@ -1738,7 +1738,7 @@ void Font_LoadOrderedFont(Font* font);
 // void func_800F50D4(void);
 void Kankyo_Init(GlobalContext* globalCtx, EnvironmentContext* envCtx);
 u32 func_800F5954(u8* param_1, u32 param_2, u32 param_3, u8 param_4, u8 param_5);
-// void func_800F5A8C(void);
+f32 func_800F5A8C(u16 param_1, u16 param_2, u16 param_3, GlobalContext* globalContext);
```

Comment:
Could you change these to use arg0, arg1, etc.? I'd like to standardise this header to only use arg over param